zhmccli/_cmd_lpar.py

```diff
@@ -519,6 +519,28 @@ def lpar_console(cmd_ctx, cpc, lpar, **options):
     cmd_ctx.execute_cmd(lambda: cmd_lpar_console(cmd_ctx, cpc, lpar, options))
 
 
+@lpar_group.command('start', options_metavar=COMMAND_OPTIONS_METAVAR)
+@click.argument('CPC', type=str, metavar='CPC')
+@click.argument('LPAR', type=str, metavar='LPAR')
+@click.option('-y', '--yes', is_flag=True, callback=abort_if_false,
+              expose_value=False,
+              help='Skip prompt to confirm starting of the LPAR.',
+              prompt='Are you sure you want to start the LPAR ?')
+@click.option('--allow-status-exceptions', is_flag=True, required=False,
+              help='Allow status "exceptions" as a valid end status.')
+@add_options(ASYNC_TIMEOUT_OPTIONS)
+@click.pass_obj
+def lpar_start(cmd_ctx, cpc, lpar, **options):
+    """
+    Start the processors from processing instructions of an LPAR.
```

Comment:
Grammar: Change to "Start the processors for processing instructions of an LPAR."

Reply:
Done